CRM/Core/Payment/PaymentExpressIPN.php

```diff
@@ -297,23 +297,33 @@ public static function main($dps_method, $rawPostData, $dps_url, $dps_user, $dps
       ));
       $success = FALSE;
       if ($response = curl_exec($curl)) {
-        fwrite($message_log, sprintf("\n\r%s:- %s\n", date("D M j G:i:s T Y"),
-          $response
-        ));
-        curl_close($curl);
-
-        // Assign the returned XML values to variables
-        $valid = CRM_Core_Payment_PaymentExpressUtils::_xmlAttribute($response, 'valid');
-        $success = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, 'Success');
-        $txnId = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, 'TxnId');
-        $responseText = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, 'ResponseText');
-        $authCode = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, 'AuthCode');
-        $DPStxnRef = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, 'DpsTxnRef');
-        $qfKey = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, "TxnData1");
-        $privateData = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, "TxnData2");
-        list($component, $paymentProcessorID,) = explode(',', CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, "TxnData3"));
-        $amount = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, "AmountSettlement");
-        $merchantReference = CRM_Core_Payment_PaymentExpressUtils::_xmlElement($response, "MerchantReference");
+        $info = curl_getinfo($curl);
+        error_log(print_r($info,1));
```

Comment:
Yup that's the debug I was referring to in [comment on PR6415](https://github.com/civicrm/civicrm-core/pull/6415#commitcomment-12531601). This commit shouldn't have gone into that branch at all, AND it shouldn't include the debug in this one. I'm off deck a few days but will put this on radar for return.